src/.gitignore

```
@@ -76,3 +76,6 @@ target/
 
 #Ipython Notebook
 .ipynb_checkpoints
+
+# mise-en-place
+.mise.toml
```

```suggestion
.mise.local.toml
.mise.*.local.toml
```

See https://mise.jdx.dev/profiles.html

---

Done :+1: